docs/content/latest/releases/versioning.md

```diff
@@ -1,15 +1,41 @@
 ---
-title: Versioning (stable and latest)
-headerTitle: What's new in 2.3
-linkTitle: What's new in 2.3
-description: Enhancements, changes, and resolved issues in the latest YugabyteDB release.
-headcontent: Features, enhancements, and resolved issues in the latest release.
-image: /images/section_icons/quick_start/install.png
-aliases:
-  - /latest/releases/
+title: Release versioning
+headerTitle: Release versioning
+linkTitle: Release versioning
+description: Explains the new release versioning approach for latest and stable releases.
 section: RELEASES
 menu:
   latest:
-    identifier: whats-new
-    weight: 2589 
----
\ No newline at end of file
+    identifier: versioning
+    weight: 2589
+isTocNested: true
+showAsideToc: true
+---
+
+Starting with version `2.2`, Yugabyte uses even a new release versioning convention for stable and development releases. The following sections covers the new release versioning convention and descriptions of stable releases and the latest release.
+
+## Release versioning convention
+
+Yugabyte follows the [semantic versioning (semver)](https://semver.org) convention for numbering release versions, modified slightly to use even and odd minor releases denote stable and development releases, respectively. Release versions follow the semantic versioning format of `MAJOR.MINOR.PATCH`, where non-negative integers are used for:
+
+- `MAJOR` — Includes substantial, potentially backward-incompatible changes.
+- `MINOR` — Incremented when new, backwards-compatible functionality is introduced.
+  - `EVEN` — Stable minor release, intended for production deployments.
+  - `ODD` — Latest minor release, intended for development and testing.
+- `PATCH` — bug fixes and revisions that do not break backwards compatibility.
+
+Examples follow in the relevant sections below.
+
+## Stable releases
+
+Stable release series, denoted by `MAJOR.EVEN` numbers, introduce fully tested new features and changes added since the last stable release and may break backwards compatibility. The current stable `2.2` release series will be followed by the next stable `2.4` release series.
```

Comment:
@stevebang This is not correct "since the last stable release and may break backward compatibility"

Reply:
Fixed.